### PR TITLE
Add dependency snippets to relevant doc sections

### PR DESCRIPTION
**Pull request by raboof**
As in #24232

### CODE REVIEW

**Comment by raboof** [comment]
Hm again `sbtwhitesource.WhiteSourceException: Error communicating with service: Unexpected error. Response data is: Bad Gateway`

**Comment by raboof** [comment]
PLS BUILD

**Comment by raboof** [comment]
PLS BUILD

**Review by johanandren — APPROVED** [review]
LGTM, thanks!

**Review by patriknw — COMMENTED** [review]
I think it's great to add this to all pages introducing a module/feature.

However, we should add it in a consistent way. Same heading, same location. 

I think it should be located as the first level 2 heading after the intro text that is typically at the top below the level 1 heading.

Heading name? "Dependency"?

File: akka-docs/src/main/paradox/cluster-sharding.md [hunk 1]
```diff
@@ -31,6 +31,16 @@ See @ref:[Downing](cluster-usage.md#automatic-vs-manual-downing).
 
 @@@
 
+## Dependency
+
+To use cluster sharding, add the module to your project:
```
Comment (patriknw) [inline akka-docs/src/main/paradox/cluster-sharding.md]: nitpick: we often write module names like Cluster Sharing

File: akka-docs/src/main/paradox/cluster-usage.md [hunk 2]
```diff
@@ -2,34 +2,15 @@
 
 For introduction to the Akka Cluster concepts please see @ref:[Cluster Specification](common/cluster.md).
 
-## Preparing Your Project for Clustering
+## Dependency
 
 The Akka cluster is a separate jar file. Make sure that you have the following dependency in your project:
```
Comment (patriknw) [inline akka-docs/src/main/paradox/cluster-usage.md]: I think the wording should be exactly the same for all these

To use Akka Cluster, add the module to your project:

**Review by patriknw — COMMENTED** [review]
consistency is king... ;-)

**Review by patriknw — APPROVED** [review]
LGTM, thanks

**Pull request merged**